docs/api/topics/addons.rst

```diff
@@ -0,0 +1,60 @@
+.. _addons:
+
+======
+Addons
+======
+
+Addon Submission
+================
+
+See the dedicated :ref:`Addon Submission <addon_submission>` topic.
+
+Addon
+=====
+
+.. note::
+
+    The `name`, field is a user-translated fields and has a dynamic type
+    depending on the query. See :ref:`translations <overview-translations>`.
+
+
+.. http:get:: /api/v2/extension/extension/
+
+    .. note:: Requires authentication.
+
+    Returns a list of addons you have submitted.
+
+    **Request**
+
+    The standard :ref:`list-query-params-label`.
+
+    **Response**
+
+    :param meta: :ref:`meta-response-label`.
+    :type meta: object
+    :param objects: A :ref:`listing <objects-response-label>` of :ref:`addons <addon-response-label>`.
+    :type objects: array
+
+.. _addon-response-label:
+
+.. http:get:: /api/v2/extensions/extension/(int:id)|(string:slug)/
```

Comment:
If we end up hosting Firefox Desktop Addons, there will be nothing left to do since they will also be extensions following the same Chrome spec (in fact, one of the big advantages of that format is having the same code for both mobile and desktop)